nxc/protocols/smb.py

```diff
@@ -591,8 +589,10 @@ def create_conn_obj(self, no_smbv1=False):
 
         :param no_smbv1: If True, it will not try to create a SMBv1 connection
         """
+        no_smbv1 = self.args.no_smbv1 if self.args.no_smbv1 else no_smbv1
+
         # Initial negotiation
-        if not no_smbv1 and self.smbv1 is None:
```

Comment:
logic from @NeffIsBack should not be changed or altered in this section

Reply:
OK